.gitignore

```diff
@@ -1,3 +1,5 @@
 log/
 tmp/
 .powenv
+.rspec
```

Comment:
what if we make `.rspec` part of the repo? I usually just have it set up to display colors. You can do that by adding this:

.
```
--color

```